hack/release/packaging/utils/create-update-packages.sh

```diff
@@ -160,26 +180,14 @@ function precheck_pub_rpms {
 # Execution of the requested steps.
 
 function docker_run_rm {
-    docker run --rm --user `id -u`:`id -g` -v $(dirname `pwd`):/code -w /code/$(basename `pwd`) "$@"
+    docker run --rm --user $(id -u):$(id -g) -v $(dirname $(pwd)):/code -w /code/$(basename $(pwd)) "$@"
 }
 
 function do_bld_images {
     # Build the docker images that we use for building for each target platform.
     pushd ${rootdir}/hack/release/packaging/docker-build-images
-    docker build -f ubuntu-trusty-build.Dockerfile.${ARCH} -t calico-build/trusty .
-    docker build -f ubuntu-xenial-build.Dockerfile.${ARCH} -t calico-build/xenial .
-    docker build -f ubuntu-bionic-build.Dockerfile.${ARCH} -t calico-build/bionic .
-    docker build -f ubuntu-focal-build.Dockerfile.${ARCH} -t calico-build/focal .
-    docker build --build-arg=UID=`id -u` --build-arg=GID=`id -g` -f centos7-build.Dockerfile.${ARCH} -t calico-build/centos7 .
+    docker buildx bake
```

Comment:
I think we've lost the passing in of `id -u` and `-id -g` in the centos7 case.  Might now be passing by accident because the current user will often be UID 1000.

Reply:
This is true, and I'll resolve this.

The centos7 case is defined in the `docker-bake.hcl` file. In that file we define a default UID/GID of 1000, which can be overridden by environment variables, but we don't override it here. I've pushed a fix for this.